### PR TITLE
Simplify case conditions in pkg/controller/statefulset/stateful_pod_c…

### PR DESCRIPTION
**Pull request by Octopusjust**
…ontrol.go

#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
The condition will be always `err = nil` after `case err != nil`
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by Octopusjust** [comment]
**cc** @krmayankk @mortent

File: pkg/controller/statefulset/stateful_pod_control.go [hunk 1]
```diff
@@ -276,7 +276,7 @@ func (spc *StatefulPodControl) PodClaimIsStale(set *apps.StatefulSet, pod *v1.Po
 			continue
 		case err != nil:
 			return false, err
-		case err == nil:
+		default:
```
Comment (yangjunmyfm192085) [inline pkg/controller/statefulset/stateful_pod_control.go:279]: /lgtm

**Review by yangjunmyfm192085 — APPROVED** [review]

**Review by yangjunmyfm192085 — COMMENTED** [review]
/assign @soltysh

**Review by soltysh — COMMENTED** [review]
Based on [the policy](https://www.kubernetes.dev/docs/guide/pull-requests/#trivial-edits) this qualifies as a trivial change that we're not going to accept. The current verbiage although verbose is probably more readable. 

/close

**Pull request closed without merging**